Sources/LiveViewNative/Stylesheets/ParseableTypes/UITextContentType+ParseableModifierValue.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  UITextContentType+ParseableModifierValue.swift
+//
+//
+//  Created by Carson Katri on 11/21/23.
+//
+
+import SwiftUI
+import LiveViewNativeStylesheet
+
+#if os(iOS) || os(tvOS)
+@available(iOS 13.0, tvOS 13.0, *)
+extension UITextContentType: ParseableModifierValue {
+    public static func parser(in context: ParseableModifierContext) -> some Parser<Substring.UTF8View, Self> {
+        ImplicitStaticMember([
+            "name": .name,
+            "namePrefix": .namePrefix,
+            "givenName": .givenName,
+            "MmddleName": .middleName,
```

Comment:
This is a typo, it should be `"middleName"`

```suggestion
            "middleName": .middleName,
```